manifest.webapp

```diff
@@ -151,35 +151,11 @@
     }
   },
   "permissions": {
-    "bank.groups": {
-      "description": "Manage groups of bank accounts",
-      "type": "io.cozy.bank.groups",
+    "banking data": {
+      "description": "Manage your banking date inside your cozy",
+      "type": "io.cozy.bank.*",
```

Comment:
I don't know how the stack & the store handle this wildcard attribute. I'm wondering if we don't need to add this "doctype" to them (https://docs.cozy.io/en/cozy-stack/doctype/#add-your-doctype-to-the-store-and-stack).

Reply:
I will remove this commit, I find it too risky at the moment.

Reply:
I used a normal permission, I am not too sure about the wildcard and I do not want to add risk right now.